src/OpenTelemetry.Instrumentation.Process/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Cleanup public API
```

Comment:
please explicitly state the API being removed, so its easy to follow changelog.

Reply:
Updated changelog.